core/src/tile/tileWorker.cpp

```diff
@@ -16,6 +16,8 @@ namespace Tangram {
 TileWorker::TileWorker(int _num_worker) {
     m_running = true;
 
+    m_mainThreadJobQueue.makeCurrentThreadTarget();
+
```

Comment:
I think it's nice to have it explicit so there's no mis-usage behavior on that. Since the default constructor could be called on other object creation holding jobQueues, it seems harder to follow.